scripts/bumpver.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+import argparse
+import subprocess
+import sys
+
+
+NAME = "logdetective"
+
+
+def call_poetry(bump_rule):
+    """Call poetry tool to bump the version."""
+    # Bump the version based on the bump rule
+    # See poetry version --help
+    subprocess.run(["poetry", "version", bump_rule], check=True)
+
+    # Read the bumped version
+    ret = subprocess.run(["poetry", "version"], capture_output=True, check=True)
+    return ret.stdout.decode().strip()
+
+
+def bump_version(target_version=None):
+    """Bump pyproject.toml version."""
+    if target_version:
+        version = call_poetry(target_version)
+    else:
+        version = call_poetry("patch")
+
+    print(f"Version bumped to {version}")
+
+    return f"{version}"
```

Comment:
Small issue. The returned string is `packagename version`. This breaks things down the line if you want to run tag.

Reply:
Good point! I even found there is `-s` option to remove all the string even from the first call so making even more optimal.